src/app/model/movie/movie.ts

```diff
@@ -0,0 +1,129 @@
+export class Movie {
+  private _id: number;
```

Comment:
in some angular component you didn't mark with underscore a private prop. No need to do that. The keyword "private" will indicate that it's private